zip/zip.go

```diff
@@ -59,3 +67,53 @@ func Zip(dirPath string, insidePath string) ([]string, []byte, error) {
 
 	return files, buf.Bytes(), nil
 }
+
+func walkSymlink(writer *zip.Writer, files *[]string, path string, destBasePath string) error {
```

Comment:
hmm shouldn't there be some recursion somewhere, maybe I missed it? what if a symlinked dir has a symlink? or we don't care about it?

Reply:
100% correct 👍 Added recursive walking in this commit: https://github.com/Tenderly/tenderly-cli/pull/136/commits/200d0d296df24c98a90c105c51ba20c161759f7e